libs/frontend/feature-build/src/components/StyleRow.vue

```diff
@@ -244,7 +244,7 @@ const removeStyle = () => {
   margin-left: 6px;
 }
 .value-input {
-  width: 140px;
+  width: 120px;
```

Comment:
This PR also included the fix for [#877](https://github.com/samatechtw/pubstudio/issues/877).

I reduced the width of value input in menu-row a little bit so that save/delete button can be seen without scrolling.

![Screenshot 2023-11-10 at 5 43 18 PM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/f6f4f59b-71a8-455d-b639-cafb36eb72cc)

Reply:
👍